lib/dirac_clover_hasenbusch_twist.cpp

```diff
@@ -29,14 +29,14 @@ namespace quda
   void DiracCloverHasenbuschTwist::M(cvector_ref<ColorSpinorField> &out, cvector_ref<const ColorSpinorField> &in) const
   {
     if (symmetric) {
-      ApplyWilsonCloverHasenbuschTwist(out[this_parity], in[other_parity], *gauge, *clover, -kappa, mu, in[this_parity],
```

Comment:
Outside of the scope of this PR, but while we're here... I see that `DiracCloverHasenbuschTwistPC` below didn't get any of the alias/etc treatment. I'm guessing there was a reason for that (unused codepath?), but if it's quick... would it be worth applying it?

Reply:
I'm not sure what you mean by the alias treatment here?  Your comment here refers to the PC class, but the line number referred to is the non PC class.

I did do some cleanup, but I don't think this is what you mean.

dcd0d439a99c31c89a6d2ec8370129df614be2af

Reply:
I have no idea what I meant either... but that clean-up looks good to me, so I'll consider today-me happy.